parts/k8s/cloud-init/artifacts/cse_install.sh

```diff
@@ -93,8 +93,8 @@ mobyPkgVersion() {
 installRunc() {
   local v
   v=$(runc --version | head -n 1 | cut -d" " -f3)
-  if [[ $v != "1.0.0-rc92" ]]; then
-    apt_get_install 20 30 120 moby-runc=1.0.0~rc92* --allow-downgrades || exit 27
```

Comment:
Notice we had a `*` here before to catch package revision.

Reply:
Got it, thanks. So to ensure we don't passively install a newer version, the literal string "1.0.0+azure" should work?

Reply:
It's a good idea (IMHO) to do `1.0.0*` because there may be tweaks to the packaging.
The full version is `1.0.0+azure-1` Where the `-1` denotes the iteration of the package.